src/coreclr/tools/aot/ILCompiler.TypeSystem/ILCompiler.TypeSystem.csproj

```diff
@@ -4,7 +4,7 @@
     <RootNamespace>Internal.TypeSystem</RootNamespace>
     <AssemblyName>ILCompiler.TypeSystem</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
We should be able to bump this to net6.0 safely. ILCompiler.Reflection.ReadyToRun has no business to depend on ILCompilerTypeSystem. I've submitted #65355 to fix that.

Reply:
I'll update if your PR is merged first, or fix this up afterwards.